### PR TITLE
Update Helm docs

### PR DESCRIPTION
**Pull request by lucacome**
The Helm docs on the website are out of date. This PR updates the docs and adds to the release scripts to automate the process. The script will copy `deployments/helm-chart/README.md` to `docs/content/installation/installation-with-helm.md` and add the necessary tags for Hugo to work.

### CODE REVIEW

File: docs/content/installation/installation-with-operator.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ docs: "DOCS-604"
 ---
 
 {{< note >}}
-An NGINX Ingress Operator version compatible with the 3.0.1 NGINX Ingress Controller release is not available yet. We will update this document and remove this note once we publish a compatible Operator version.
+An NGINX Ingress Operator version compatible with the 3.0.2 NGINX Ingress Controller release is not available yet. We will update this document and remove this note once we publish a compatible Operator version.
```
[the remark this answers is not in the document]
Comment (lucacome) [reply docs/content/installation/installation-with-operator.md]: This file was modified by mistake.

**Comment by ciarams87** [comment]
Looks good! Does this mean that we will only have to modify `deployments/helm-chart/README.md` for Helm changes going forward? In other words, this will be the source of truth and so we should _not_ modify `docs/content/installation/installation-with-helm.md` manually any more?

**Comment by lucacome** [comment]
@ciarams87 yep! `deployments/helm-chart/README.md` will be the source of truth and everything will be copied over to `docs/content/installation/installation-with-helm.md`

**Review by shaun-nx — APPROVED** [review]

File: deployments/helm-chart/README.md [hunk 2]
```diff
@@ -113,6 +84,19 @@ The command removes all the Kubernetes components associated with the release an
 
 Uninstalling the release does not remove the CRDs. To remove the CRDs, see [Uninstalling the CRDs](#uninstalling-the-crds).
 
+### Edge Version
+
+To test the latest changes in the Ingress Controller before a new release, you can install the `edge` version. This version is built from the `main` branch of the Ingress Controller repository.
```
Comment (ADubhlaoich) [inline deployments/helm-chart/README.md:89]:
```suggestion
To test the latest changes in NGINX Ingress Controller before a new release, you can install the `edge` version. This version is built from the `main` branch of the NGINX Ingress Controller repository.
```

**Review by ciarams87 — APPROVED** [review]

**Pull request merged**